File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer =''
    for change_s in s :
        step = 0
        while(step!=index):
            change_s = chr(ord(change_s)+1)
            if change_s=="{" : change_s = 'a'
            
            if change_s in skip:
                continue
            else:
                step +=1
        answer += change_s
    return answer

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    var answer = '';
    
    
    for(let char of s){
        let count = 0;
        let code = char.charCodeAt(0); 
        
        while(count < index){
            code++;
            if(code > 'z'.charCodeAt(0)) code = 97 // 'a'
            
            const nextChar = String.fromCharCode(code);
            if(!skip.includes(nextChar)) count++;
        }
        answer += String.fromCharCode(code)
    }
    
    return answer;
}